bc/assets/templates/includes/footer.html

```diff
@@ -0,0 +1,122 @@
+{% load static %}
+
+<section class="mt-16">
+    <div class="mx-auto px-4 sm:px-6 md:px-10">
+        <section id="footer"class="grid grid-cols-1 xs:grid-cols-2 sm:grid-cols-3 md:grid-cols-4 gap-8 md:gap-10 lg:gap-12 w-full py-8 justify-start">
+          <div class="w-full hidden md:block row-span-1">
+            <img src="{% static "svg/icon.svg" %}"  layout="responsive" class="rounded-lg border-4 border-black h-36"/>
```

Comment:
I think you'll see that nothing is black in this design. It's generally good practice not to have black stuff, but instead to have nearly-black stuff. In this case I'm using #1A1A1A, I think, instead of black. 

I'd suggest we add it to our tailwind.config as a special black color. We should do the same for the Yellow that we use. Check out how it's done in the free.law project.

Reply:
Alright!. I will customize the `theme` section of the `tailwind.config.js` file.

I found this website: https://uicolors.app/create to create Tailwind CSS color shades based on a given hexcode